$NetBSD: patch-CMakeLists.txt,v 1.4 2026/03/25 23:01:03 wiz Exp $

We don't want the package finding ccache because pkgsrc handles
ccache on its own.

--- CMakeLists.txt.orig	2026-01-29 12:48:17.000000000 +0000
+++ CMakeLists.txt
@@ -110,7 +110,6 @@ if(LLVM_SPIRV_BUILD_EXTERNAL)
   endif()
 
   option(CCACHE_ALLOWED "allow use of ccache" TRUE)
-  find_program(CCACHE_EXE_FOUND ccache)
   if(CCACHE_EXE_FOUND AND CCACHE_ALLOWED)
     message(STATUS "Found ccache: ${CCACHE_EXE_FOUND}")
     set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
